client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/style.scss

```diff
@@ -29,6 +29,43 @@
 	}
 }
 
+.launchpad__container {
+	display: flex;
+	flex-direction: column;
+}
+
+.launchpad__email-validation-banner {
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	padding: 16px;
+	background: var(--studio-gray-0);
+
+	&-content {
+		gap: 16px;
+		display: flex;
+		align-items: center;
+
+		&-resend-button {
```

Comment:
[QUESTION] This may be more a question for Design, but do you know if we have any additional hover styles we are using for the buttons and links in our new banner?

Reply:
Good question which I don't know the answer to (cc @nuriapenya )

Reply:
+1 - It feels a bit odd clicking the `Resend email` link and `change email address` link without this styling. Not a blocker for the time being for me since functionally everything works well.